### PR TITLE
`archive-fork-links` - Update Selector

### PR DESCRIPTION
**Pull request by kovsu**
close: #7334 

## Test URLs

https://github.com/dnSpy/dnSpy

## Screenshot

image

image

### CODE REVIEW

File: source/features/archive-forks-link.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ function addLinkToBanner(banner: HTMLElement): void {
 }
 
 function init(signal: AbortSignal): void {
-	observe('#js-repo-pjax-container > .flash-warn:first-child', addLinkToBanner, {signal});
+	observe('#js-repo-pjax-container > .flash-warn', addLinkToBanner, {signal});
```
Comment (fregante) [inline source/features/archive-forks-link.tsx:19]: The problem is that sometimes there's more than one banner I think. Can you `git blame` the line to find out when that happens? Maybe `:first-of-type` still works?

Comment (kovsu) [reply source/features/archive-forks-link.tsx:19]: `:first-of-type` works

**Review by fregante — APPROVED** [review]
Thank you!

**Comment by FloEdelmann** [comment]
There is a typo in the PR title and in the changelog:

```diff
- archive-fork-links
+ archive-forks-link
```

**Pull request merged**